indexer/services/comlink/__tests__/controllers/api/v4/transfers-controller.test.ts

```diff
@@ -287,5 +287,223 @@ describe('transfers-controller#V4', () => {
         ],
       });
     });
+
+    it('Get /transfers/parentSubaccountNumber returns transfers/deposits/withdrawals', async () => {
```

Comment:
nit: Can you include 1 transfer that is sent to a child subaccount (the isolated subaccount) and check that it exists in the response?

Reply:
In this [thread](https://dydx-team.slack.com/archives/C06ETJJGYLX/p1711037888343109), I thought we decided that we'd be filtering out any transfers from parent <> child SAs on the indexer?

I have another test which tests exclusion of these internal transfers in this PR